src/VueModel.js

```diff
@@ -188,11 +188,11 @@ class VueModel {
         });
 
         Vue.models = {
-            register: this.register,
+            register: this.register.bind(this),
```

Comment:
Is this what was causing the problems using this package?

Reply:
Yes. I was able to create a simple demo after adding this fix.